lib/omedis_web/live/event_live/index.ex

```diff
@@ -82,13 +83,28 @@ defmodule OmedisWeb.EventLive.Index do
       )
       |> Ash.load(:group, authorize?: false)
 
+    if connected?(socket) do
+      :ok = Endpoint.subscribe("#{activity.id}:events")
+    end
+
     {:noreply,
      socket
      |> assign(:activity, activity)
      |> assign(:group, activity.group)
      |> apply_action(socket.assigns.live_action, params)}
   end
 
+  @impl true
+  def handle_info(%Phoenix.Socket.Broadcast{event: "create"} = broadcast, socket) do
+    new_event = Map.get(broadcast.payload, :data)
+    {:noreply, stream_insert(socket, :events, new_event)}
+  end
+
+  def handle_info(%Phoenix.Socket.Broadcast{event: "update"} = broadcast, socket) do
+    updated_event = Map.get(broadcast.payload, :data)
+    {:noreply, stream_insert(socket, :events, updated_event)}
+  end
```

Comment:
These two seem pretty similar. You can make it DRY:
```suggestion
  def handle_info(%Phoenix.Socket.Broadcast{event: event} = broadcast, socket) when event in ["create", "update"] do
    event = Map.get(broadcast.payload, :data)
    {:noreply, stream_insert(socket, :events, event)}
  end
```

Either remove the `@impl` or put `@impl Phoenix.LiveView` (remember Credo rules we've had in other projects?)